store/postgres/migrations/2022-04-26-125552_alter_deployment_schemas_version/up.sql

```diff
@@ -0,0 +1,62 @@
+--  Alter `deployment_schemas.version` from enum to integer.
+--  Some views are recreated because they depend on the altered column.
+
+-- This is left for the fdw logic which runs after migrations to recreate.
+drop schema if exists primary_public cascade;
+
+drop view if exists info.subgraph_info;
+drop view if exists info.all_sizes;
+drop materialized view if exists info.subgraph_sizes;
+
+alter table if exists deployment_schemas alter version type integer using 0;
```

Comment:
I wouldn't say `if exists` here - if that table doesn't exist something is seriously wrong, and we should fail the migration.